### PR TITLE
Fix typos in comments and tests

### PR DESCRIPTION
**Pull request by alexandear**
### What's being changed:

This PR corrects spelling mistakes.

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

**Comment by alexandear** [comment]
Agree with the CLA

**Comment by parkerduckworth** [comment]
Hey @alexandear thanks for the PR! I reran the integration tests after the first fail to see if it was just a flake, but looks like there might actually be something wrong here.

Can you investigate when you have a chance? When that is resolved, we can proceed with a merge.

**Comment by alexandear** [comment]
@parkerduckworth fixed integration tests

File: adapters/repos/db/filters_integration_test.go [hunk 1]
```diff
@@ -977,7 +977,7 @@ var cars = []models.Object{
 			"horsepower":             int64(100),
 			"weight":                 1200.0,
 			"contact":                "sandra@efficientcars.example.com",
-			"description":            "This small car has a small engine and unicode labels (ąę), but it's very light, so it feels faster than it is.",
+			"description":            "This small car has a small engine and unicode labels (ąę), but it's very light, so it feels fater than it is.",
```
Comment (parkerduckworth) [inline adapters/repos/db/filters_integration_test.go:980]: You should be able to actually change this back to `faster`. I also investigated the reason for the failing test, and it's due [this filter](https://github.com/weaviate/weaviate/blob/master/adapters/repos/db/filters_integration_test.go#L523-L526) which expects the description of `carPolo` to match a specific length.

Adding the `s` to `fater` exceeds this expected length, so the id is not returned in the search. You can simply change this expected description length from `109` to `110`, and the test should pass. This way we can eliminate this typo as well 🙂

**Review by parkerduckworth — CHANGES_REQUESTED** [review]

File: adapters/repos/db/filters_integration_test.go (hunk 1 above)
Comment (alexandear) [reply adapters/repos/db/filters_integration_test.go]: Thanks.

File: usecases/auth/authentication/oidc/middleware.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/weaviate/weaviate/usecases/config"
 )
 
-// Client handles the OIDC setup at startup and provides a middleware to tbe
+// Client handles the OIDC setup at startup and provides a middleware to the
```
Comment (parkerduckworth) [inline usecases/auth/authentication/oidc/middleware.go:25]: Judging by the rest of the comment, this looks like it should be `tbe` -> `be`

Comment (alexandear) [reply usecases/auth/authentication/oidc/middleware.go]: Agree.

**Review by parkerduckworth — APPROVED** [review]

**Comment by parkerduckworth** [comment]
Thanks @alexandear

**Pull request merged**